crates/bevy_ui/src/widget/image.rs

```diff
@@ -3,16 +3,15 @@ use bevy_asset::{Assets, Handle};
 use bevy_color::Color;
 use bevy_ecs::prelude::*;
 use bevy_image::Image;
+use bevy_image::TRANSPARENT_IMAGE_HANDLE;
 use bevy_math::{Rect, UVec2, Vec2};
 use bevy_reflect::{std_traits::ReflectDefault, Reflect};
-use bevy_render::texture::TRANSPARENT_IMAGE_HANDLE;
 use bevy_sprite::{TextureAtlas, TextureAtlasLayout, TextureSlicer};
-use bevy_window::{PrimaryWindow, Window};
+use bevy_window::PrimaryWindow;
+use bevy_window::Window;
 use taffy::{MaybeMath, MaybeResolve};
-
 /// A UI Node that renders an image.
 #[derive(Component, Clone, Debug, Reflect)]
-#[reflect(Component, Default, Debug)]
```

Comment:
Is there a reason reflection was removed here?

Reply:
That looks like an error on my part. There should be a test to catch such a big change, tsk tsk tsk.